app/repositories/eventrepository.php

```diff
@@ -100,4 +101,56 @@ public function getEventsByDetailPageId(int $id)
             echo $e;
         }
     }
+    public function addEvent(Event $event)
+    {
```

Comment:
should be in a try catch, there is always something that can go wring with databases